src/main/java/edu/cornell/kfs/fp/batch/service/impl/AdvanceDepositServiceImpl.java

```diff
@@ -186,16 +198,31 @@ public boolean routeAdvanceDepositDocuments() {
                     LOG.info("Routing advance deposit document # " + documentId + ".");
                 }
                 documentService.routeDocument(advanceDocument, "document routed by achIncome batch job", null);
-            } catch (WorkflowException e) {
-                LOG.error("Error routing document # " + documentId + " " + e.getMessage());
-                throw new RuntimeException(e.getMessage(), e);
+            }
+            catch (Exception e) {
+                LOG.error("Error routing document # " + documentId + " due to exception: " + e.getMessage());
+                logException(e);
             }
         }
 
         return true;
+    }  
+
+    private void logException(Exception e) {
+        if (e instanceof ValidationException) {
+            List<String> errors = GlobalVariablesUtils.extractGlobalVariableErrors();
+            if (ObjectUtils.isNotNull(errors) && !errors.isEmpty()) {
+                for (String errorMessage : errors) {
+                    LOG.error(errorMessage);
+                }
+            }
+        } else {
+            LOG.error(e.getStackTrace());
+        }
     }
+      
 
-    /**
+	/**
      * Returns a list of all initiated but not yet routed advance deposit documents, using the WorkflowDocumentService.
```

Comment:
Small formatting issue. Looks like tabs versus spaces. Will not hold up PR if there are no other issues that need fixing.